ui/app/components/tool-unwrap.js

```diff
@@ -16,14 +16,12 @@ import { action } from '@ember/object';
  *  @onClear={{action "onClear"}}
  *  @token={{token}}
  *  @unwrap_data={{unwrap_data}}
- *  @unwrapActiveTab={{unwrapActiveTab}}
  *  @details={{details}}
  *  @errors={{errors}}/>
  * ```
  * @param onClear {Function} - parent action that is passed through. Must be passed as {{action "onClear"}}
  * @param token=null {String} - property passed from parent to child and then passed back up to parent
  * @param unwrap_data {String} - property returned from parent.
- * @param unwrapActiveTab {String} - property returned from parent.
```

Comment:
Was this just a documented prop that was unused?

Reply:
I replaced the button + nav in this component to use the HDS tabs (same as I did with the policy template above) to remove instances `is-active` classes applied to buttons